Lib/pdb.py

```diff
@@ -1414,7 +1412,6 @@ def do_break(self, arg, temporary=False):
     # To be overridden in derived debuggers
     def defaultFile(self):
         """Produce a reasonable default."""
-        assert self.curframe is not None
```

Comment:
This should have been replaced with the check you did. Could you make your changes more carefully?

Reply:
> This was meant to be added in completedefault not here. This function must check that curframe exists.

Reply:
sorry for my English will check the code first next time, not language part
will changet it carefully 
Thanks again